README.md

```diff
@@ -57,11 +57,8 @@ If adding a camera which does not needed mapped because the focus point is given
 
 Known Issues
 --------
-1. Not currently working on Windows. 
```

Comment:
As such, you can not find a rotation to solve a mirroring. In the *particular* case of our current icons which are symmetrical, we *could* make a calculation to find a fitting rotated icon to match a mirrored one.
I dont like particular cases in programming so I went the way to not take this particular case into account and created mirrored icons.